src/PKSim.Core/Repositories/IContainerParametersNotCommonForAllSpeciesRepository.cs

```diff
@@ -0,0 +1,10 @@
+﻿using System;
+using OSPSuite.Utility.Collections;
+
+namespace PKSim.Core.Repositories
+{
+   public interface IContainerParametersNotCommonForAllSpeciesRepository : IStartableRepository<(string ContainerPath, string ParameterName, int SpeciesCount)>
```

Comment:
This is the repo which retrieves the info, if a parameter is used by all species or not.
I did not defined a new typo for the repo objects, because they are probably not of interest.
The only important thing is if a parameter in a container is used by all species or not.
So I just defined a named tuple instead `(string ContainerPath, string ParameterName, int SpeciesCount)`

Reply:
hum....this is not what we do normally and just for consistency sake, I would use a type here.

Reply:
ok, can change it, np